ast/compile.go

```diff
@@ -86,6 +86,9 @@ type Compiler struct {
 	// with the key being the generated name and value being the original.
 	RewrittenVars map[Var]Var
 
+	// Capabilities is the user-supplied capabilities or features allowed for OPA.
+	Capabilities *Capabilities
```

Comment:
Exposing this technically means it could be messed with. However, `WithCapabilities()` already allows that, so I guess we're OK. Alternatively, we could do
```go
func (c *Compiler) Capabilities() *Capabilities {
    return c.capabilities
}
```

Reply:
There already is precedence for direct access for other fields, but if a function is preferred, I can add one. The only drawback would be that it might look haphazard to a user.

Reply:
I've been thinking about this, too, it's the only part of this PR that worries me. Once we expose this as a field, we can never go back or change or structures or something.

Having a getter function would leave us done wiggle room. I'd prefer we go with that.

Reply:
Alright! Will fix.